seed/building_sync/building_sync.py

```diff
@@ -42,6 +42,7 @@ class ParsingError(Exception):
 
 class BuildingSync(object):
     BUILDINGSYNC_V2_0 = '2.0'
+    BUILDINGSYNC_V2_2_0 = '2.2.0'
```

Comment:
Had to add the 2.2.0 BSync schema for testing our generated documents were valid